src/main/java/seedu/cakecollate/model/orderitem/Type.java

```diff
@@ -0,0 +1,56 @@
+package seedu.cakecollate.model.orderitem;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.cakecollate.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents the type of cake of an order item.
+ * Guarantees: immutable; is valid as declared in {@link #isValidType(String)}
+ */
+public class Type {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Order Type should only contain alphabets and white spaces, and it should not be blank";
+
+    /**not sure if there is a way to simplify this regex? It matches anything that
+     * contains only alphabets and does not start or end with a white space*/
+    public static final String VALIDATION_REGEX = "^([a-zA-Z]|([a-zA-Z]+[a-zA-Z\\s]*[a-zA-Z]))$";
+
+    public final String value;
```

Comment:
I am not too sure what the variable value means here.

Reply:
I followed the same convention as the Person model where they use value to store the value of the object.